src/content/app-platform/create-catalog/index.md

```diff
@@ -0,0 +1,164 @@
+CRs.
+
+There are [multiple options](https://helm.sh/docs/topics/chart_repository/#hosting-chart-repositories)
+for serving the catalog over HTTP including GitHub Pages or tools like Harbor or ChartMuseum which run in a
+Kubernetes cluster. At Giant Swarm we use GitHub Pages and this is what we will
+cover in this guide.
+
+## Creating an app catalog hosted using GitHub Pages
+
+First, you should choose a name for your catalog. We recommend using the suffix
+`-catalog` to make it clear this Git repository hosts an app catalog.
+
+Create the Git repository in GitHub as a public repository and enable
+[GitHub Pages](https://docs.github.com/en/pages/quickstart) for the `main` branch.
+
+## Publishing an app to the app catalog
+
+Now you can configure your apps to be published to your catalog. We have created
+a GitHub Action for `app-build-suite` that will automate this.
+
+```yaml
+# .github/workflows/push-to-app-catalog.yaml
+name: 'Push to App Catalog'
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  push_to_app_catalog:
+    uses: giantswarm/app-build-suite/.github/workflows/push-to-app-catalog.yaml@v1.1.0
+    with:
+      app_catalog: example-catalog
+      chart: hello-world-app
+      organization: example-org
+    secrets:
+      envPAT: ${{ secrets.PAT }}
+```
+
+To configure the action you need to add a secret called `PAT` to the app's git
+repository. This must contain a [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+that has write permission to the app catalog git repository.
+
+We recommend using [Dependabot](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/keeping-your-actions-up-to-date-with-dependabot)
+to keep the GitHub Action up to date.
+
+## Create Catalog CR
+
+Once you've created your app catalog and published some apps you need to create
+a Catalog CR in the management cluster to register it with app platform. This
+can be done using [kubectl gs template catalog]({{< relref "/ui-api/kubectl-gs/template-catalog" >}}).
+
+You should create the Catalog CR in the [organization]({{< relref "/general/organizations" >}})
+namespace where the apps will be used.
+
+```nohighlight
+kubectl gs template catalog \
+  --name example \
+  --namespace org-example \
+  --description "An example Catalog" \
+  --url https://example.github.io/example-catalog/ \
+  --logo https://example.com/logos/example-logo.png
+```
+
+```yaml
+apiVersion: application.giantswarm.io/v1alpha1
+kind: Catalog
+metadata:
+  creationTimestamp: null
+  name: example
+  namespace: org-example
+spec:
+  description: An example Catalog
+  logoURL: https://example.com/logos/example-logo.png
+  storage:
+    URL: https://example.github.io/example-catalog/
+    type: helm
+  title: example
+```
+
+## Referencing an app catalog in an App CR
```

Comment:
Again, the headline should be more goal or use case oriented. In this case, I think it means "Installing an app from the catalog to a workload cluster", right?

Reply:
Yes, this is for installing the app.